src/online.jl

```diff
@@ -1,40 +1,69 @@
 import Base: append!, push!
 
-function push!(di::LinearInterpolation{U, T}, u::eltype(U), t::eltype(T)) where {U, T}
-    push!(di.u, u)
-    push!(di.t, t)
-    di
+function push!(A::LinearInterpolation{U, T}, u::eltype(U), t::eltype(T)) where {U, T}
+    !A.safetycopy && throw(UnsafeAddDataError())
```

Comment:
Is this needed? Shouldn't we allow for both?

Reply:
That is fine by me, I think it is clear enough that this leads to mutation of the input `u, t` if someone explicitly sets `safetycopy = false`.